identifiers/country-us/state-ma-executive-offices.csv

```diff
@@ -0,0 +1,9 @@
+id,name
+ocd-division/country:us/state:ma/governors_council:1,Massachusetts Governor's Council 1st District
```

Comment:
From a quick look at wikipedia, this sounds similar to New Hampshire's Executive Council (names seem to be interchangable between the two):
https://en.wikipedia.org/wiki/Massachusetts_Governor%27s_Council
https://en.wikipedia.org/wiki/Executive_Council_of_New_Hampshire

For consitency let's use "executive_council" as the type instead of "governors_council"

Reply:
All set 👍